src/cloud-code/Parse.Cloud.js

```diff
@@ -637,6 +667,54 @@ module.exports = ParseCloud;
  * @property {Object} params The params passed to the cloud function.
  */
 
+/**
+ * @interface Parse.Cloud.ValidatorFunction
+ * @property {String} installationId If set, the installationId triggering the request.
+ * @property {Boolean} master If true, means the master key was used.
+ * @property {Parse.User} user If set, the user that made the request.
+ * @property {Object} params The params passed to the cloud function.
+ */
+
+/**
+ * @interface Parse.Cloud.ValidatorObject
+ * @property {Boolean} requireUser whether the cloud trigger requires a user.
+ * @property {Boolean} requireMaster whether the cloud trigger requires a master key.
+ * @property {Object|Array<String>} fields if an array of strings, validator will look for keys in request.params, and throw if not provided. If Object, {@link Parse.Cloud.ValidatorFields} to validate. If the trigger is a cloud function, `request.params` will be validated, otherwise `request.object`.
```

Comment:
This is confusing. Can you move the `ValidatorFields` here? `@property` should be a description of the property only.

Reply:
How is that now @dplewis?